test/cfg/boost.cpp

```diff
@@ -33,6 +40,8 @@ void valid_code()
 
     int int1 = 5;
     boost::endian::endian_reverse_inplace(int1);
+    boost::bind(print_hello)();
```

Comment:
see https://github.com/danmar/cppcheck/pull/2207#issuecomment-534117757 !?

Reply:
There is and should be no warning for this line.
The return value of `boost::bind()` is directly used to call the function. (see the `()` behind the function call).
This is not possible with `std::bind()` as far as i know.

Reply:
My mistake